docs/about/change_log.rst

```diff
@@ -0,0 +1,107 @@
+.. _change-log:
+
+===================
+PlasmaPy Change Log
+===================
+
+This document provides a detailed list of changes associated with each
+release of PlasmaPy.  The release_notes_ contain a narrative description
+for each version.
+
+.. _change-log-0.1.0:
+
+Version `0.1.0` (2018-04-27)
+--------------------------
+
+Version `0.1.0` is the initial development release of PlasmaPy.  This
+version is a prototype and a preview, and is not feature complete.
+Significant changes to the API are expected to occur between versions
+`0.1.0` and `0.2.0`.
+
+.. _change-log-0.1.0-new:
+
+New Features
+~~~~~~~~~~~~
+
+* The vision statement describes the motivation of early PlasmaPy
+  developers in creating a fully open source Python package for plasma
+  physics.
+
+* Added a code of conduct.
+
+* Added a contribution guide.
+
+* Adopted a BSD 3-clause license and added protections against
+  software patents.
+
+* Set up continuous integration testing with Travis CI, CircleCI, and
+  AppVeyor, along with test coverage checks with Coveralls.
+
+* Decided upon code and docstring style conventions and set up
+  automated code style checks with pep8speaks.
+
+* Set up automated documentation builds with Sphinx that are hosted on
+  Read the Docs.
+
+* Adopted use of `~astropy.units` as a units package.  Allowed import of
+  units as ``import plasmapy.units as u``.
```

Comment:
I am still not sure this last point is a good idea. In fact I may have acted on that removed this from init in a commit to one of your recent PRs... I'm not sure, I'll have to check.